src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -485,6 +487,21 @@ public void startActivity(Intent intent) {
     }
   }
 
+  @SuppressLint("RestrictedApi")
+  @Override
+  public boolean onCreateOptionsMenu(Menu menu){
+
+    MenuInflater inflater = this.getMenuInflater();
+    inflater.inflate(R.menu.conversation, menu);
+
+    if(menu instanceof MenuBuilder){
```

Comment:
Is there a better way to do this? I just find type checking is awful haha but if there is no other way thats fine.

Reply:
Unfortunately no, because  you need to do dynamic casting, you have to check before hand or risk throwing errors.